src/output/plugins/httpd/HttpdClient.cxx

```diff
@@ -102,6 +112,15 @@ HttpdClient::HandleLine(const char *line) noexcept
 			return true;
 		}
 
+		if (StringEqualsCaseASCII(line, "Authorization: Basic ", 21)) {
+			auto encoded_pwd_length = strlen(line) - 21;
+			size_t debase64_size = CalculateBase64OutputSize(encoded_pwd_length);
+			auto debase64_buffer = std::make_unique<char[]>(debase64_size);
+			debase64_size = DecodeBase64({debase64_buffer.get(), debase64_size}, line + 21);
+			std::string auth_str = std::string(debase64_buffer.get(), debase64_size);
```

Comment:
Why copy the string into a newly allocated `std::string` here? Operating on a `std::string_view` would eliminate this heap allocation + copy.

Reply:
Done.